src/actions/index.js

```diff
@@ -118,3 +123,29 @@ export function fetchProjects(parms) {
       })
   }
 }
+
+
+export function fetchPublications() {
+  return dispatch => {
+    map((key) => {
+      addIndex(forEach)(
+        (project, idx) => {
+          dispatch(setState(`publications.${key}.projects.${idx}.publications`, project.publications))
+          dispatch(setState(`publications.${key}.projects.${idx}.slug`, project.slug))
+
+          if (project.slug) {
```

Comment:
This uses the link (project slug) to fetch the avatar and display name of each project.  the project slug is hardcoded along with each of the publications. (I hope that answers the question?)